### PR TITLE
Detail for kraj and okres

### PR DESCRIPTION
**Pull request by jan-stanek**

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -22,7 +19,7 @@ def info_mista():
     mista = db.session.query(OckovaciMisto.id, OckovaciMisto.nazev, Okres.nazev.label("okres"),
                              Kraj.nazev.label("kraj"), OckovaciMistoMetriky.registrace_fronta,
                              OckovaciMistoMetriky.registrace_prumer_cekani, OckovaciMistoMetriky.ockovani_odhad_cekani) \
-        .join(OckovaciMistoMetriky, (OckovaciMistoMetriky.id == OckovaciMisto.id)) \
+        .join(OckovaciMistoMetriky) \
```
Comment (msusicky) [inline app/views.py:22]: Maybe better to show which attribute is used...

**Review by msusicky — CHANGES_REQUESTED** [review]
Metrics are crashing if no data there -> better just warn, or?
 File "W:\Personal\COVID19\ockovani\ockovani-covid\app\etl.py", line 686, in _find_import_id
    raise Exception("No data for date: '{0}'.".format(self._date))

**Review by msusicky — CHANGES_REQUESTED** [review]
And we have to think how to put queue info to the kraje overview (request from Klatovy)

File: app/views.py (hunk 1 above)
Comment (jan-stanek) [reply app/views.py:22]: ETL crash if data are not present is intention, there should be separated task to improve ETL to be able to reconstruct historical data where possible (before registrations and reservations were available)

**Review by msusicky — APPROVED** [review]

**Pull request merged**